assets/js/components/Work/Tabs/About.jsx

```diff
@@ -108,12 +109,14 @@ const WorkTabsAbout = ({ work }) => {
     // with React Hook Form's register().   So, we'll use getValues() to get the real data
     // updated.
     let currentFormValues = methods.getValues();
+
     const { title = "" } = currentFormValues;
     let workUpdateInput = {
       descriptiveMetadata: {
         alternateTitle: prepFieldArrayItemsForPost(
           currentFormValues.alternateTitle
         ),
+        dateCreated: currentFormValues.dateCreated || [],
```

Comment:
Here we want to pass to `prepFieldArrayItemsForPost` function to convert from an array of objects to array of strings.